src/common-ssh/ssh.c

```diff
@@ -416,7 +427,7 @@ static int guac_common_ssh_authenticate(guac_common_ssh_session* common_session)
 
 guac_common_ssh_session* guac_common_ssh_create_session(guac_client* client,
         const char* hostname, const char* port, guac_common_ssh_user* user, int keepalive,
-        const char* host_key) {
+        const char* host_key, guac_ssh_credential_handler* credential_handler) {
```

Comment:
This is called `credential_callback` in the prototype. Either is fine with me, but the prototype and function body should be consistent.

Reply:
Went with `credential_handler` across the board.